pkg/server/user.go

```diff
@@ -96,22 +108,16 @@ func (s *ZomatoServer) PlaceOrder(ctx context.Context, in *pb.OrderRequest) (*pb
 	//extract his id (agent_id)
 	//get the id of the user who is calling this method (user_id)
 	//create a new order with the agent_id and user_id and some restaurant_id
-	totalAgentData := []*pb.Agent{}
+	totalActiveAgentData := []*pb.Agent{}
```

Comment:
why do you need this protobuf struct again? you can use model struct directly right?